.travis.yml

```diff
@@ -1,24 +1,25 @@
 language: ruby
 
-rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.5
-  - 2.2.0
-  - ruby-head
-  - jruby-19mode
-  - jruby-head
+sudo: false
 
-jdk:
-  - oraclejdk7
-  - openjdk7
-  - oraclejdk8
+before_install:
+  - gem update --system
+  - gem update bundler
```

Comment:
We might not need this anymore if the latest ruby in the 2.5 series on Travis is 2.5.1.